### PR TITLE
[launch] add batch task helpers

### PR DESCRIPTION
**Pull request by syandroo**
add option to pass in args instead of url

### CODE REVIEW

File: launch/client.py [hunk 1]
```diff
@@ -832,6 +835,7 @@ def batch_async_request(
         self,
         bundle_name: str,
         urls: List[str],
+        inputs: Optional[List[Dict[str, Any]]],
```
Comment (seanshi-scale) [inline launch/client.py:838]: should we do `urls: Optional[List[str]] = None` and `inputs: Optional[...] = None`?

File: launch/client.py [hunk 2]
```diff
@@ -875,8 +880,15 @@ def batch_async_request(
                 f"Disallowed options {set(batch_task_options.keys()) - allowed_batch_task_options} for batch task"
             )
 
+        if not (bool(inputs) ^ bool(urls)):
```
Comment (seanshi-scale) [inline launch/client.py:883]: Also what are people's opinions on allowing a mix of urls/inputs for batch inference? I definitely don't think we need to do it now, but I'm ambivalent on in the future. Wonder if it makes sense to even have a mix of urls + inputs.

**Review by seanshi-scale — APPROVED** [review]
Had some nits but otherwise lgtm

File: launch/client.py (hunk 1 above)
Comment (syandroo) [reply launch/client.py:838]: omg yea my b

File: launch/client.py (hunk 2 above)
Comment (yixu34) [reply launch/client.py]: I think I'd lean towards not allowing the mix. Seems esoteric IMO.

**Pull request merged**